src/App.css

```diff
@@ -1,39 +1,194 @@
-.App {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Great job so far @katfogy 🤝

- [ ] As stated in the project requirements. Kindly ensure your UI design is a replica of the design that was given. Kindly ensure the margins, color, and spacing of all the elements are an imitation of the Zeplin design given. 
- Your design
![your design](https://github.com/katfogy/bookstore-react/assets/63917725/7460ba73-aa98-415b-848f-09d312457b67)
- Given design
![Given design](https://github.com/katfogy/bookstore-react/assets/63917725/d01d1ff1-e882-41c4-841a-d8fd8cf7a962)
You may also check out this [link](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png) to look at the design provided.